src/Particle.js

```diff
@@ -2099,6 +2103,29 @@ class Particle {
 		});
 	}
 
+	/**
+	 * Set default auth token that will be used in each method if `auth` is not provided
+	 * @param  {String} auth A Particle access token
+	 * @returns {undefined}
+	 */
+	setDefaultAuth(auth){
+		this._defaultAuth = auth;
```

Comment:
if you are worried about accepting invalid values - e.g. `true`, `{}`, etc. - i would validate here. it's basically "must be non-empty string", correct?

Reply:
Sure, 'tiz an easy change, conquered in https://github.com/particle-iot/particle-api-js/pull/145/commits/bc940ab6d22998c60d9fc3de3d061b1138327167